WTWSMS/decisions/conversion_decisions.txt

```diff
@@ -533,10 +537,7 @@ targetted_decisions = {
 						NOT = { religion_group = ROOT }
 					}
 				}
-				random = {
-					chance = 60
-					add_trait = sympathy_christendom
-				}
+				add_trait = sympathy_christendom
```

Comment:
That's true. Thus there should be a chance of adding Zealotry as well, to negate it.